src/payloads/payload.ts

```diff
@@ -248,10 +250,13 @@ export const transactionIdResponseSecond = {
 export const getPaymentRequestsGetResponse = (
   senderService: ServicePublic
 ): PaymentRequestsGetResponse => ({
-  importoSingoloVersamento: faker.datatype.number({
-    min: 1,
-    max: 9999
-  }) as PaymentRequestsGetResponse["importoSingoloVersamento"],
+  importoSingoloVersamento:
+    (ioDevServerConfig.wallet.payment
+      .amount as PaymentRequestsGetResponse["importoSingoloVersamento"]) ??
+    (faker.datatype.number({
+      min: 1,
+      max: 9999
+    }) as PaymentRequestsGetResponse["importoSingoloVersamento"]),
```

Comment:
```suggestion
  getRandomValue(
    ioDevServerConfig.wallet.payment?.amount,
    faker.datatype.number({
      min: 1,
      max: 9999
    }),
    "wallet"
  ) as ImportoEuroCents,
```
you could use `getRandomValue` function to take advantage of randomness control (for each section)
If you find it as a good idea, [this function](https://github.com/pagopa/io-dev-api-server/blob/1a047b5f5c88ebf8c2720380fc5ec453fb771f4a/src/utils/random.ts#L13) should be changed

from this
```
const getValueGlobalRandomOn: RandomValueFunc = <T>(
  defaultValue: T,
  randomValue: T,
  configSectionKey: AllorRandomValueKeys
) =>
  ioDevServerConfig[configSectionKey].allowRandomValues
    ? randomValue
    : defaultValue;
```
to this
```typescript
const getValueGlobalRandomOn: RandomValueFunc = <T>(
  defaultValue: T,
  randomValue: T,
  configSectionKey: AllorRandomValueKeys
): T =>
  ioDevServerConfig[configSectionKey].allowRandomValues ||
  defaultValue === undefined
    ? randomValue
    : defaultValue;
```

Reply:
Addressed in 1f8270f